### PR TITLE
Update Bond to 7.2.0

### PR DESCRIPTION
**Pull request by knutnyg**
I've updated Bond to 7.2.0

## Description

The migration was a bit tricky, bit not very invasive. I had most trouble with some helper functions previously available being removed. 

## Motivation and Context
Solves: https://github.com/LN-Zap/zap-iOS/issues/73

## How Has This Been Tested?
Runs on my phone ;)

## Checklist:

- [X] My code follows the code style of this project
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: Library/Scenes/ConnectRemoteNode/ConnectRemoteNodeViewModel.swift [hunk 1]
```diff
@@ -112,15 +104,22 @@ final class ConnectRemoteNodeViewModel: NSObject {
...
-        
+                certificate: remoteNodeConfiguration.certificate,
+                macaroon: remoteNodeConfiguration.macaroon,
+                url: url)
+
         self.remoteNodeConfiguration = newConfiguration
     }
 }
+
+public func to2DArraySection<T, U>(section: T, items: [U]) -> TreeNode<Array2DElement<T, U>> {
```
Comment (knutnyg) [inline Library/Scenes/ConnectRemoteNode/ConnectRemoteNodeViewModel.swift]: I'm not sure where this should live. It's not an extension, more like a utility function

**Comment by ottosuess** [comment]
awesome!! i'll take a look

**Comment by knutnyg** [comment]
I'll fix the whitespace changes so it can be merged

**Comment by knutnyg** [comment]
I’ll reopen when it’s stable

**Pull request closed without merging**